readme.md

```diff
@@ -16,7 +16,7 @@ Enjoy!
 
 - Get familiar with [the Pomodoro Technique](http://pomodorotechnique.com/) (read the [paper](http://baomee.info/pdf/technique/1.pdf))
 - Read the [Manifesto for Agile Software Development](http://www.agilemanifesto.org/)
-- Read the [Principles behind the Agile Manifesto](http://www.agilemanifesto.org/principles.html)
+- Read the [Principles behind the Agile Manifesto](http://agilemanifesto.org/)
```

Comment:
And here we want to keep the link to the _Principles behind the Agile Manifesto_. Consider to remove the `www` part only.

Reply:
Yap, I did it in a hurry, will fix now :/